### DISCUSSION TITLE
Is it a bad idea to give users permission to UPDATE themselves?

### DISCUSSION THREAD

**Discussion by Xyphyn** [pocketbase/pocketbase#1944] [Q&A] [answered]
The thing I'm most worried about is them having the ability to just verify themselves. Is this an issue?

**Comment by ganigeorgiev** [comment, marked as answer]
The `verified` field cannot be updated directly by regular users unless they have "Manage API rule" access.

**Reply by Xyphyn** [reply]
Thanks!
